src/Repository/InvoiceRepository.php

```diff
@@ -84,4 +86,13 @@ public function getTotalPriceOfAllInvoices(): float {
 
         return $result !== null ? (float) $result : 0.0;
     }
+
+    public function findByStatus(string $typePayment): array
+    {
+        return $this->createQueryBuilder('i')
+            ->andWhere('i.typePayment = :typePayment')
+            ->setParameter('typePayment', $typePayment)
+            ->getQuery()
+            ->getResult();
+    }
```

Comment:
Please add a //TODO 

I think we will change this after implemeting the status handling

The name of the function is findByStatus but you are looking from the payment type

Reply:
i can just change the name of the function and make findByTypePayment

Reply:
Yes we could but in any case, we should add the findByStatus in second time